feature-pack/build.xml

```diff
@@ -7,13 +7,13 @@
                 <filter token="infinispan.brand.version" value="${infinispan.brand.version}"/>
                 <filter token="infinispan.module.slot" value="${infinispan.module.slot}"/>
                 <filter token="infinispan.module.slot.prefix" value="${infinispan.module.slot.prefix}"/>
-                <filter token="lucene.module.slot" value="${version.lucene.module.slot}"/>
                 <filter token="hibernate.search.module.slot" value="${version.hibernate.search.module.slot}"/>
                 <filter token="gson.module.slot" value="${version.gson.module.slot}"/>
             </filterset>
             <regexpmapper from="(.*)/slot/(.*)" to="\1/${infinispan.module.slot}/\2" />
         </move>
         <replace dir="${feature-pack.modules}" includes="**" token="@hibernate.search.module.slot@" value="${version.hibernate.search.module.slot}"/>
+        <replace dir="${feature-pack.modules}" includes="**" token="@lucene.module.slot@" value="${version.lucene.module.slot}"/>
```

Comment:
Any idea why `move` works for the other tokens? Maybe we should use `replace` for all of them just to be sure?

Reply:
No I don't, so I agree it's probably best to migrate all of them to `replace`